installation/install

```diff
@@ -264,16 +283,24 @@ handle_error $?
 
 script_header "INSTALLATION ET CONFIGURATION DE ZSH ET OH MY ZSH"
 
-script_header "ZSH" $C_YELLOW
-sys_install zsh
+if "$os" != "solus"; then
+	script_header "ZSH" $C_YELLOW
+	sys_install zsh
+fi
 
-script_header "OH-MY-ZSH" $C_YELLOW
-sh -c "$(curl -fsSL https://raw.github.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"
-#no error check here: remote install.sh has a chsh error
-chsh $USER -s /usr/bin/zsh
-handle_error $?
-echo "alias blih='blih -u $1'" >> /home/$USER/.zshrc
-echo "alias ns_auth='ns_auth -u $1'" >> /home/$USER/.zshrc
+if "$os" != "solus"; then
+	script_header "OH-MY-ZSH" $C_YELLOW
+	sh -c "$(curl -fsSL https://raw.github.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"
+	sudo chsh $USER -s /bin/zsh
+	handle_error $?
+	echo "alias blih='blih -u $1'" >> /home/$USER/.zshrc
+	echo "alias ns_auth='ns_auth -u $1'" >> /home/$USER/.zshrc
+	echo "alias ne='emacs -nw'" >> /home/$USER/.zshrc
+fi
+
+echo "alias blih='blih -u $1'" >> /home/$USER/.bashrc
+echo "alias ns_auth='ns_auth -u $1'" >> /home/$USER/.bashrc
+echo "alias ne='emacs -nw'" >> /home/$USER/.bashrc
```

Comment:
to reduce code duplication, maybe you could populate a `~/.aliases.sh` file with aliases, and source it in `~/.bashrc` & `~/.zshrc`